gcn/notices/icecube/LVK_nu_track_search.example.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "type": "IceCube LVK Alert Nu Track Search",
+  "ref_ID": ["MS230427r"],
```

Comment:
```suggestion
  "ref_ID": "MS230427r",
```

Reply:
thanks!  wonder how that pass schema verification...